adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/WEB-INF/views/responsive/pages/checkout/multi/checkoutSummaryPage.jsp

```diff
@@ -27,13 +27,19 @@
                 var amountJS = {value: "${amount.value}", currency: "${amount.currency}"};
                 AdyenCheckoutHybris.initiatePaypal(amountJS, "${immediateCapture}", "${paypalMerchantId}", "hidden-xs");
                 AdyenCheckoutHybris.initiatePaypal(amountJS, "${immediateCapture}", "${paypalMerchantId}", "visible-xs");
-        </c:when>
+            </c:when>
 
             <c:when test="${selectedPaymentMethod eq 'mbway'}">
                 AdyenCheckoutHybris.initiateMbway("hidden-xs");
                 AdyenCheckoutHybris.initiateMbway("visible-xs");
             </c:when>
 
+            <c:when test="${selectedPaymentMethod eq 'applepay'}">
+                var amountJS = {value: "${amount.value}", currency: "${amount.currency}"};
+                AdyenCheckoutHybris.initiateApple(amountJS, "${countryCode}", "${applePayMerchantIdentifier}", "${applePayMerchantName}", "hidden-xs");
```

Comment:
all other components are using "locale". why do we specifically need to pass countrycode here?

Reply:
It is required by Apple Pay component.
https://docs.adyen.com/payment-methods/apple-pay/web-component#page-introduction